apps/site/pages/docs/getting-started-glossary.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: 'Glossary'
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+<header>
+
+# Glossary
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+  This documentation is currently under development.
+</Callout>
+
+
+## C 
+
+### Content Type
+The nature of pages created via the [VTEX Headless CMS](/docs/headless-cms-overview). 
+A Landing Page, a Product Listing Page (PLP), and a Product Detail Page (PDP) are a few examples of different Content Types.
+
+---
+
+## D 
+
+### Design tokens
+Design elements that define the behavior and appearance of components in a store. 
+There are two types: [global tokens](https://www.faststore.dev/docs/global-tokens/overview), which control system-wide interactions, and [local tokens](/docs/theming-overview#global-and-local-tokens), which customize individual components. 
+By configuring both types, consistent visual styles can be maintained while allowing for customization at different levels.
+
+
+---
+
+## F
+
+### FastStore Onboarding 
+The app that simplifies the initial setup of your FastStore project.
+With this app, you can easily configure essential store settings such as Catalog, CMS integration, and SEO tags. 
+
+### FastStore CLI
+The Command Line Interface (CLI) for initializing, building and developing FastStore projects.
+
+---
+
+## N
+
+### Next.js
+A [Static Site Generator](/docs/getting-started-glossary#static-site-generator-ssg) created by Vercel for developing React-based web applications server-side rendering and generating static websites.
+
+---
+
+## O
+
+### Override
+A method to modify a component's props or replace a native component with a customized version to achieve the desired behavior or functionality.
+
+---
+
+## S
+
+### Sections
+React components that wrap up other components, facilitating the creation of cohesive and functional content for your storefront. For example, the [Hero](/components/organisms/hero) section. In the context of the [Headless CMS](/docs/headless-cms-overview), sections are predefined and customizable content structures. They act as building blocks that can be populated with content, such as text, images, and media, allowing users to publish content in a more user-friendly and standardized manner. The integration of sections with a CMS streamlines the content creation process and ensures a consistent look and feel across different pages or sections of storefront. 
```

Comment:
I'm wondering where we should link the Hero section, maybe to the section listed [here](https://faststore-site-git-docs-onboarding-dashboard-faststore.vercel.app/docs/reference-guides/list-of-native-sections)?